src/router.ts

```diff
@@ -180,6 +180,7 @@ class Router {
       for (const dipDeposit of Object.values(this.dips)) {
         if (dipDeposit.qtyTokens > 0) {
           openPositionCount++;
+          // TODO await for a response back from the API
```

Comment:
Any time we route an order successfully, we dupe the run_risk_manager. 1. From CheckMM prices & 2. From position change callback. The position change lags sufficiently where it is picked up after we've awaited on routing.

Either we need something to turn off the callback temporarily or some other fix with separate routing logic for re-runs vs position change. Or whatever you think is a good close solution